db/migrations/0002_user_order_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,96 @@
+# Generated by Django 4.0.2 on 2024-11-21 17:23
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',

```

Comment:
The `User` model is being created here. If this is intended to replace Django's default user model, make sure that `AUTH_USER_MODEL` is set to this model in your Django settings.